src/main.jsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
-import './index.css'
```

Comment:
Include the reset css file here so you can chop off the `margin: 0;`'s from the `App.css` file.